components/CustomerCard.js

```diff
@@ -0,0 +1,40 @@
+import Image from "next/image";
+import { IoIosQuote } from "react-icons/io";
+
+export default function CustomerCard() {
+  const data = {
+    id: "1",
+    imgSrc: "/../public/avatar.png",
+    name: "Sammy Lawson",
+    position: "Creative Director, Amazon",
+    feedback:
+      "White dwarf a still more glorious dawn awaits tingling of the spine   emerged into consciousness Vangelis shores of the cosmic ocean. Tendrils of gossamer clouds kindling the energy hidden in matter   concept of the number one permanence.",
+  };
+
+  return (
+    <div>
+      <div className="bg-white text-gray-500 	rounded-xl p-8 shadow-lg m-4">
```

Comment:
text colors aren't all gray-500. Name is `text-black` and position and text is `#64758D`